ppr-ui/src/enums/transferTypes.ts

```diff
@@ -1,17 +1,61 @@
 export enum ApiTransferTypes {
+  // Bill Of Sale Transfers
   SALE_OR_GIFT = 'TRANS',
+  TRANS_FAMILY_ACT = 'TRANS_FAMILY_ACT',
+  TRANS_INFORMAL_SALE = 'TRANS_INFORMAL_SALE',
+  TRANS_QUIT_CLAIM = 'TRANS_QUIT_CLAIM',
+  TRANS_RECEIVERSHIP = 'TRANS_RECEIVERSHIP',
+  TRANS_SEVER_GRANT = 'TRANS_SEVER_GRANT',
+  TRANS_WRIT_POSSESSION = 'TRANS_WRIT_POSSESSION',
+
+  // Transfers Due to Death
   SURVIVING_JOINT_TENANT = 'TRAND',
   TO_EXECUTOR_UNDER_25K_WILL = 'TRANS_AFFIDAVIT',
   TO_ADMIN_NO_WILL = 'TRANS_ADMIN',
-  TO_EXECUTOR_PROBATE_WILL = 'TRANS_WILL'
+  TO_EXECUTOR_PROBATE_WILL = 'TRANS_WILL',
+
+  // Other Transfers
+  ABAN = 'ABAN',
```

Comment:
I pulled these from the spreadsheet, but I can also update them to something more meaningful. LMK!